CHANGELOG.md

```diff
@@ -8,6 +8,12 @@ The format is based on [Keep a Changelog] and this project adheres to [Semantic
 
 * None
 
+## [Version 0.7.1] - [Unreleased]
```

Comment:
If this is a link then the URL needs adding at the bottom.

But actually I usually save the version bump until we're ready to do a release, so you can just mark this section Unreleased and remove the change to Cargo.toml.

Reply:
Thanks!!